src/Core/Other.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * @param {(Component|Element|DOM|Node)} element
+ * @return {Node?}
+ */
+function findDOMNode (element) {
+	if (element) {
+		if (DOMValid(element))
+			return element
+
+		if (DOMValid(element.target))
```

Comment:
is this part of jsdom? where is target set?

Reply:
In src/DOM/DOM.js#5